storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/KvStoreAccessor.java

```diff
@@ -82,8 +94,31 @@
 
     <T> void put(KvStoreVariable<T> variable, T value);
 
+    /**
+     * Write raw bytes to a specified variable WARNING: should only be used to migrate data between
+     * database instances
```

Comment:
```suggestion
     * Write raw bytes to a specified variable.
     *
     * <p>WARNING: should only be used to migrate data between
     * database instances
```

and same change below so the warning is a bit more obvious (this is guaranteed to fail spotless as-is)